openqa-investigate

```diff
@@ -374,7 +374,14 @@ investigate() {
     comment_id=$(sync_via_investigation_comment "$id" "$first_cluster_job_id") || rc=$?
     [[ $rc != 255 ]] && return $rc
 
-    out=$(trigger_jobs "$id" "${@:2}") || return $?
+    out=""
+    if ! out=$(trigger_jobs "$id" "${@:2}"); then
+        local comment="Triggering investigation jobs failed.
+        Check journal for reasons."
```

Comment:
```suggestion
        local comment="Triggering investigation jobs failed. Check journal for reasons."
```

Reply:
I've included the stderr both on the job comment and as output of the script